addons/website/views/snippets/s_cta_card.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_cta_card" name="Card Call to Action">
+    <section class="s_cta_card o_cc o_cc2 pt64 pb64" data-oe-shape-data="{'shape':'web_editor/Blocks/03'}">
```

Comment:
Ensure that the shape (`Blobs_12`, see my previous comment) is visible on mobile 😉